tag_game_presets.json

```diff
@@ -850,6 +850,265 @@
+                "Fungus1_23": [
+                    "left1"
+                ]
+            },
+            "min_players": 8,
+            "max_players": 15,
+            "transitions": {
+                "Fungus3_13": [
+                    "right1",
+                    "bot1"
+                ]
+            },
+            "loadouts": {
+                "normal": {
+                    "charms": [
+                        "WaywardCompass", "Kingsoul", "Dashmaster", "SporeShroom"
+                    ],
+                    "skills": [
+                        "CycloneSlash", "DashSlash", "GreatSlash", "MantisClaw", "CrystalHeart", "MonarchWings",
+                        "IsmasTear", "ShadeCloak", "MothwingCloak", "VengefulSpirit"
+                    ]
+                },
+                "infected": {
+                    "charms": [
+                        "NailmastersGlory", "Dashmaster", "Sprintmaster", "SharpShadow"
+                    ],
+                    "skills": [
+                        "CycloneSlash", "DashSlash", "GreatSlash", "MantisClaw", "CrystalHeart", "MonarchWings",
+                        "IsmasTear", "ShadeCloak", "MothwingCloak"
+                    ]
+                }
+            }
+        },
+        {
+            "name": "queens-gardens-petra-small",
+            "warp_transitions": {
+                "Fungus3_08": [
+                    "right1"
+                ]
+            },
+            "min_players": 3,
+            "max_players": 6,
+            "transitions": {
+                "Fungus3_08": [
+                    "right1",
+                    "left1",
+                    "top1"
+                ]
+            },
+            "loadouts": {
+                "normal": {
+                    "charms": [
+                        "WaywardCompass", "Kingsoul", "Dashmaster", "Flukenest", "SharpShadow"
+                    ],
+                    "skills": [
+                        "CycloneSlash", "DashSlash", "GreatSlash", "MantisClaw", "CrystalHeart", "MonarchWings",
+                        "IsmasTear", "ShadeCloak", "MothwingCloak", "VengefulSpirit"
+                    ]
+                },
+                "infected": {
+                    "charms": [
+                        "NailmastersGlory", "Dashmaster", "Sprintmaster"
+                    ],
+                    "skills": [
+                        "CycloneSlash", "DashSlash", "GreatSlash", "MantisClaw", "CrystalHeart", "MonarchWings",
+                        "IsmasTear", "ShadeCloak", "MothwingCloak"
+                    ]
+                }
+            }
+        },
+        {
+            "name": "queens-gardens-petra-medium",
+            "warp_transitions": {
+                "Fungus3_08": [
+                    "left1"
+                ],
+                "Fungus3_10": [
+                    "top1"
+                ],
+                "Fungus3_39": [
+                    "left1"
+                ]
+            },
+            },

```

Comment:
There is a duplicate `},` here, which invalidates the JSON file. If you put the entire file into your HKMP.Tag mod folder and boot up the game, you can figure out if the JSON and schema is incorrect.